test/format.test.js

```diff
@@ -14,86 +14,100 @@ const options = {
 let logger;
 let spy;
 
-function testLog(testOptions, testSpy) {
-    logger.format = configuredFormatter({}, testOptions);
-    logger.error('message');
+function testLoggerInfo(testOptions, testSpy) {
+    logger.format = configuredFormatter(testOptions);
+    logger.info('message', { foo: 'bar' });
     expect(testSpy).toHaveBeenCalled();
 }
 
 beforeEach(() => {
     logger = createLogger({
-        level: 'silly'
+        level: 'silly',
+        transports: [
+            new winston.transports.Console({ silent: true })
+        ]
     });
 });
 
 describe('configuredFormatter testing', () => {
     describe('console output', () => {
         beforeEach(() => {
-            // spy to check output
             spy = jest.fn((info) => {
                 const sym = Symbol.for('message');
-                const testMess = info[sym];
-                expect(testMess).toEqual(`${info.timestamp} ${info.level}: ${info.message}`);
+                const testMsg = info[sym];
+                expect(testMsg).toEqual(`${info.timestamp} ${info.level}: ${info.message}`);
             });
             logger.add(new winston.transports.SpyTransport({ spy }));
         });
         test('it passes console format check', (done) => {
             options.typeFormat = 'console';
-            testLog(options, spy);
+            testLoggerInfo(options, spy);
             done();
         });
         test('it checks NODE_ENV for development and sets to console', (done) => {
             process.env.NODE_ENV = 'dev';
-            testLog(options, spy);
+            testLoggerInfo(options, spy);
             done();
         });
     });
-    describe('json output', () => {
+
+    describe('json output (info)', () => {
         beforeEach(() => {
-            // spy to check output
             spy = jest.fn((info) => {
+                expect(spy).toHaveBeenCalled();
```

Comment:
`testLoggerInfo` is also calling this check. Remove from here and leave in `testLoggerInfo`